client/components/gsuite/gsuite-new-user-list/new-user.tsx

```diff
@@ -87,8 +99,8 @@ const GSuiteNewUser: FunctionComponent< Props > = ( {
 						setMailBoxFieldTouched( wasValidated );
 					} }
 					onKeyUp={ onReturnKeyPress }
-					prefix={ isRtl ? `\u200e@${ domain }\u202c` : null }
-					suffix={ isRtl ? null : `\u200e@${ domain }\u202c` }
+					prefix={ isRtl ? `\u200e@${ selectedDomain.name }\u202c` : null }
+					suffix={ isRtl ? null : `\u200e@${ selectedDomain.name }\u202c` }
```

Comment:
Do we need to handle undefined values like this?

```suggestion
					prefix={ isRtl ? `\u200e@${ selectedDomain?.name }\u202c` : null }
					suffix={ isRtl ? null : `\u200e@${ selectedDomain?.name }\u202c` }
```

Reply:
You are right, I had to do a manual merge for isRtl changes and I missed it.